chapters/en/Unit 1 - Fundamentals/Chapter 1 - Fundamentals of Computer Vision/applications.mdx

```diff
@@ -32,99 +36,36 @@ Medical image analysis involves the application of computer vision and machine l
+| Accuracy and Accountability	          | Computer vision systems must be accurate and reliable. Accountability measures are necessary to address errors or failures, ensuring that those responsible for system development are held accountable for any unintended consequences.                                                      |
+| Consent and Informed Decision Making	 | Obtaining informed consent from individuals whose data is being collected or used by computer vision systems is essential. Users should be informed about how their data will be used and have the right to make informed decisions about its usage.                                          |
+| Dual Use Concerns	                    | Computer vision technology can have both beneficial and potentially harmful uses. Ensuring that the technology is not used for malicious purposes, such as surveillance or invasion of privacy, is crucial.                                                                                   |
+| Transparency and Explainability	      | Computer vision systems should be transparent in their functioning and decisions. Users should be able to understand how these systems work and the reasons behind their decisions.                                                                                                           |
+| Child Protection	                     | Special care must be taken when dealing with visual data involving children. Safeguards should be in place to protect minors from privacy violations or any other potential harm.                                                                                                             |
+| Cultural and Contextual Sensitivity	  | Computer vision systems should be sensitive to cultural differences and diverse contexts to avoid misinterpretations or biases based on cultural or regional norms.                                                                                                                           |
+| Human Oversight	                      | Human oversight and intervention are crucial in ensuring that computer vision systems operate ethically and make accurate decisions. Humans should have the ability to intervene in cases where the system's decisions might cause harm.                                                      |
+| Environmental Impact                  | The development and deployment of computer vision systems should consider their environmental impact. This includes energy consumption, electronic waste, and other ecological factors.                                                                                                       |
+| Educational and Ethical Training	     | Training programs and educational initiatives are essential to raise awareness about the ethical implications of computer vision technology among developers, users, and policymakers.                                                                                                        |

```

Comment:
There is this concept called "less is more", although we got around 10 things in our table few seems really off so i propose trimming to 5-7 with few paragraphs etc above the table but love to get your ideas as well. Eitherway we should work on re-doing few bits here.

Reply:
What about making subtopics? 

Consent and Informed Decision Making and Child Protection can be subtopics of privacy concerns

Impact of model can regroup Dual Use Concerns, Transparency and Explainability, and Environmental Impact.

Educational and Ethical Training + Cultural and Contextual Sensitivit can be a subtopic of accountability